api/src/main/java/org/openmrs/module/pihcore/setup/MetadataMappingsSetup.java

```diff
@@ -80,6 +80,11 @@ public static void setupPrimaryIdentifierType(MetadataMappingService metadataMap
         metadataMappingService.mapMetadataItem(patientIdentifierType, EmrApiConstants.EMR_CONCEPT_SOURCE_NAME, EmrApiConstants.PRIMARY_IDENTIFIER_TYPE);
     }
 
+    public static void setupPrimaryIdentifierType(MetadataMappingService metadataMappingService, PatientService patientService, String identifierTypeUuid) {
+        PatientIdentifierType patientIdentifierType = patientService.getPatientIdentifierTypeByUuid(identifierTypeUuid);
+        metadataMappingService.mapMetadataItem(patientIdentifierType, EmrApiConstants.EMR_CONCEPT_SOURCE_NAME, EmrApiConstants.PRIMARY_IDENTIFIER_TYPE);
+    }
```

Comment:
Maybe I should have looked at the CES config PR closer, but it this removed because Iniz supports setting up metadata mappings (great, if so!)

Reply:
Nothing is removed.  I just added a new method that takes the uuid rather than the descriptor, as the descriptor went away and the uuid is available.

Reply:
Ah, got it, missed that...